ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -293,7 +304,9 @@ IndexValue find(StoreKey keyToFind) throws StoreException {
               high = mid - 1;
             }
           }
-          metrics.bloomFalsePositiveCount.inc(1);
+          if (bloomPositive) {
+            metrics.bloomFalsePositiveCount.inc(1);
+          }
         }
         return null;
```

Comment:
This looks correct. I just feel that this method is a good example of how multiple return points affect readability. Since we are anyway refactoring this method, do you think you can fix that?